matching-core/src/main/java/com/graphhopper/matching/util/HmmProbabilities.java

```diff
@@ -67,33 +55,12 @@ public double emissionLogProbability(double distance) {
      *                       consecutive map matching candidates.
      * @param linearDistance Linear distance [m] between two consecutive GPS
      *                       measurements.
-     * @param timeDiff       time difference [s] between two consecutive GPS
-     *                       measurements.
      */
-    public double transitionLogProbability(double routeLength, double linearDistance,
-                                           double timeDiff) {
-        if (timeDiff == 0) {
```

Comment:
> because penalties from unfavored edges would still result in a transition probability of 1

@stefanholder would you mind to explain this to me? Do we just pick the distance then? But is this not somehow conflicting with the FastestWeighting we use?